Runner.py

```diff
@@ -0,0 +1,31 @@
+"""
+Оформите решение задач из прошлых домашних работ в функции.
+Напишите функцию runner.
+(все станет проще когда мы изучим модули, getattr и setattr)
+a.	runner() – все фукнции вызываются по очереди
+b.	runner(‘func_name’) – вызывается только функцию func_name.
+c.	runner(‘func’, ‘func1’...) - вызывает все переданные функции
+
+"""
+import funcs
+
+
+def runner(*args):
+    # print(*args)
+    if len(args) == 0:
+        for name in dir(funcs):
+            if '__' not in name:
+                f = getattr(funcs, name)
+                f()
+    else:
+        # print(args)
+        for arg in args:
+            if arg in dir(funcs):
```

Comment:
__ check?

Reply:
if arg in dir(funcs): 
this code checks for the existence of a called function in the "funcs" module,
if we pass wrong arguments to the runner, the execution will follow the path:
             else:
                 print ("Unknown function")

additionally, a check is made for possible names __name__ in the module

Reply:
we shouldn't give the ability to run __ functions from module.